cwd/app.js

```diff
@@ -269,16 +271,18 @@
        * This makes it so that the factory cannot work to add data to gauges.
        * Instead, we need to add the data while handling the view object.
        */
+      let resStatus = 0;
       if (obj.view && obj.view.gauges) {
         obj.view.gauges.forEach((g, index) => {
-          if (g.data_url) {
+           if (g.data_url) {
             fetch(g.data_url)
-            .then(r => r.json())
+            .then(r => {
+              r.json();
+            })
```

Comment:
I *think* this needs to return something or else the later code will break.  Not sure.